script/script.go

```diff
@@ -189,18 +198,19 @@ func (s *script) Response(f filters.FilterContext) {
 func (s *script) runFunc(name string, f filters.FilterContext) {
 	L, err := s.getState()
 	if err != nil {
-		log.Printf("ERROR: %s", err)
+		log.Errorf("ERROR: %v", err)
```

Comment:
I guess `ERROR:` we don't need and better to add more context where we are, such that users can understand that this is likely a lua script `getState` error (whatever the error message will look like